bin/scripts/docker-entrypoint.sh

```diff
@@ -23,6 +23,6 @@ done
 printf "Running with options:\n%s\n" "$args"
 
 # shellcheck disable=SC2086
-for f in /in/*.otf /in/*.ttf /in/*.woff /in/*.eot /in/*.ttc; do [ -f "$f" ] && fontforge -script /nerd/font-patcher -out /out $args "$f"; done
+find /in -type f \( -name "*.otf" -o -name "*.ttf" -o -name "*.woff" -o -name "*.eot" -o -name "*.ttc" \) | parallel fontforge -script /nerd/font-patcher -out /out $args {}
```

Comment:
I like `find` much more than the loop. We could use `-iname` here, as some fonts out there have their name and extension in caps. (Which would be an additional commit of course ;)

Reply:
Probably I would use `-iregex` but that is because I'm a regex person and hate the `-o` style in `find` ;-)

Reply:
```suggestion
find /in -type f -iregex ".*\.\(otf|ttf|woff|eot|ttc\)" | parallel fontforge -script /nerd/font-patcher -out /out $args {}
```
_untested_

Reply:
Dockerfile's `find` is a BusyBox multi-call, which is not support `-iregex`, I will change it with `-iname`.